packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -45,6 +46,11 @@ export async function ensureExtensionsIds(
     dashboardOnlyExtensions,
     validIdentifiers,
   )
+  const adminLinkExtensionsToMigrate = getAdminLinkExtensionsToMigrate(
```

Comment:
I noticed that all `get**ExtensionsToMigrate` functions do mostly the same, do you want to unify that in a generic function? :) 
We can leave it for a future PR if not